AP-protocol.md

```diff
@@ -307,30 +307,42 @@ TODO
 
 # Simulation Controller
 
-Like nodes, the **Simulation Controller** runs on a thread. It must retain a means of communication with all nodes of the network, even when drones go down.
+Like nodes, the **Simulation Controller** (SC) runs on a thread. It must retain a means of communication with all nodes of the network, even when drones go down. 
+The Simulation controller can send and receive different commands to/from the nodes (drones, clients and servers) through reserved channels. The list of available **commands** is as follows:
 
-### Simulation commands
-
-The Simulation Controller can send the following commands to drones:
-
-`Crash`: This commands makes a drone crash. Upon receiving this command, the drone’s thread should return as soon as possible.
+```rust
+pub enum Command {
+    AddChannel(NodeId, Sender<Packet>),
+    RemoveChannel(NodeId),
+    Crash,
+    ChangePdr(f32),
+    SentMessage(Packet),
+    DroppedMessage(Packet)
+}
```

Comment:
I would split in two enum

Reply:
I don't think we need to split the enum for now, maybe later in the implementation, but since it needs to be a unique type to be sent in the same channel, for now this doesn't add too much complexity.

Reply:
The point is that if we split in two we don't have to check extra illegal cases.

Becuase we would have a channel of commandFromSC e un channel of commandFromNode (names are very indicatives)

Reply:
You would just need to add a _ in the match case. I get your point but I don't really see the trade-off with having even more channels to manage.